queries.sql

```diff
@@ -71,4 +71,10 @@ INNER JOIN vets v ON v.id = vet_id WHERE v.id = 2;
 SELECT species.name, COUNT(*) FROM species
 INNER JOIN animals ON species.id = animals.species_id
 INNER JOIN visits ON animals.id = animal_id
-INNER JOIN vets v ON v.id = vet_id WHERE v.id = 2 GROUP BY species.name ORDER BY count DESC;
\ No newline at end of file
+INNER JOIN vets v ON v.id = vet_id WHERE v.id = 2 GROUP BY species.name ORDER BY count DESC;
+
+
+
+EXPLAIN ANALYZE SELECT COUNT(*) FROM visits where animal_id = 4;
+EXPLAIN ANALYZE SELECT * FROM visits where vet_id = 2;
```

Comment:
- [ ] Kindly you should do as much as possible to normalize your database performance by reducing the execution time after improving performance.  the project request that you should get execution time after normalization which is under 1 sec (1000ms).

your execution time(after improving performance):
![image](https://user-images.githubusercontent.com/90524466/185951642-6de75b4d-9bfa-47aa-980e-1dc198d9d9f1.png)

Reply:
## Improvement of performance
![Capture](https://user-images.githubusercontent.com/102048113/185965108-59785ab6-64fc-4a79-b886-1899e037a5c4.PNG)